README.rst

```diff
@@ -46,6 +46,7 @@ The various geocoder classes are located in `geopy.geocoders`_.
 .. _SmartyStreets: https://smartystreets.com/products/liveaddress-api
 .. _GeocodeFarm: https://www.geocodefarm.com/
 .. _Here: https://developer.here.com/documentation/geocoder/
+.. _MapBox: https://www.mapbox.com/api-documentation/#geocoding
```

Comment:
In RST this is just a named link, which should be referenced somewhere. More specifically, it must be added to the paragraph above this list.